fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/OnboardingPage.kt

```diff
@@ -66,12 +69,15 @@ private const val URL_TAG = "URL_TAG"
  * @param modifier The modifier to be applied to the Composable.
  * @param onDismiss Invoked when the user clicks the close button. This defaults to null. When null,
  * it doesn't show the close button.
+ * @param imageIsIllustration True if the image is an illustration, otherwise false.
  */
 @Composable
+@Suppress("LongMethod")
 fun OnboardingPage(
     pageState: OnboardingPageState,
     modifier: Modifier = Modifier,
     onDismiss: (() -> Unit)? = null,
+    imageIsIllustration: Boolean,
```

Comment:
I think we can set a reasonable default value here, based on the param change it could be `true` or `Fit` if we go the enum route

Reply:
imo I think we should rely on Nimbus to provide this value, inc. the default. We _should_ never have a scenario where this value isn't provided, wdyt?